src/scalar_planner_estimates.rs

```diff
@@ -0,0 +1,220 @@
+}
+
+// --- End code generated by tools/estimates/fit_butterflies.py ---
+
+// --- Begin code generated by tools/estimates/fit_radix4.py ---
+
+const RADIX4_CONST: f32 = 0.90900;
+const RADIX4_SLOPE: f32 = 0.58761;
+const RADIX4_EXP: f32 = 1.15980;
+
+pub fn estimate_radix4_cost(len: usize, repeats: usize) -> f32 {
+    (RADIX4_CONST + RADIX4_SLOPE * (len as f32).powf(RADIX4_EXP)) * repeats as f32
+}
+
+// --- End code generated by tools/estimates/fit_radix4.py ---
+
+// --- Begin code generated by tools/estimates/fit_raders.py ---
+
+const RADERS_CONST: f32 = -1.22742;
+const RADERS_SLOPE: f32 = 2.30372;
+
+pub fn estimate_raders_cost(inner_fft: &Arc<Recipe>, repeats: usize) -> f32 {
+    (RADERS_CONST + RADERS_SLOPE * (inner_fft.len() + 1) as f32 + 2.0 * inner_fft.cost(1))
+        * repeats as f32
+}
+
+// --- End code generated by tools/estimates/fit_raders.py ---
+
+// --- Begin code generated by tools/estimates/fit_bluesteins.py ---
+
+const BLUESTEINS_CONST: f32 = 16.77562;
+const BLUESTEINS_LEN_SLOPE: f32 = 0.30714;
+const BLUESTEINS_INNER_LEN_SLOPE: f32 = 0.34986;
+
+pub fn estimate_bluesteins_cost(len: usize, inner_fft: &Arc<Recipe>, repeats: usize) -> f32 {
+    (BLUESTEINS_CONST
+        + BLUESTEINS_INNER_LEN_SLOPE * inner_fft.len() as f32
+        + BLUESTEINS_LEN_SLOPE * len as f32
+        + 2.0 * inner_fft.cost(1))
+        * repeats as f32
+}
+
+// --- End code generated by tools/estimates/fit_bluesteins.py ---
+
+// --- Begin code generated by tools/estimates/fit_mixedradixes.py ---
+
+const MIXEDRADIX_SLOPE: f32 = 0.09496;
+const MIXEDRADIX_EXP: f32 = 1.18301;
+
+pub fn estimate_mixedradix_cost(
+    len: usize,
+    left_fft: &Arc<Recipe>,
+    right_fft: &Arc<Recipe>,
+    repeats: usize,
+) -> f32 {
+    let inners_cost = left_fft.cost(right_fft.len()) + right_fft.cost(left_fft.len());
+    let twiddle_cost = MIXEDRADIX_SLOPE * (len as f32).powf(MIXEDRADIX_EXP);
+    (inners_cost + twiddle_cost) * repeats as f32
+}
+
+const MIXEDRADIXSMALL_SLOPE: f32 = 1.07425;
+const MIXEDRADIXSMALL_EXP: f32 = 0.86855;
+
+pub fn estimate_mixedradixsmall_cost(
+    len: usize,
+    left_fft: &Arc<Recipe>,
+    right_fft: &Arc<Recipe>,
+    repeats: usize,
+) -> f32 {
+    let inners_cost = left_fft.cost(right_fft.len()) + right_fft.cost(left_fft.len());
+    let twiddle_cost = MIXEDRADIXSMALL_SLOPE * (len as f32).powf(MIXEDRADIXSMALL_EXP);
+    (inners_cost + twiddle_cost) * repeats as f32
+}
+
+const GOODTHOMAS_SLOPE: f32 = 2.45554;
+const GOODTHOMAS_EXP: f32 = 0.98744;
+
+pub fn estimate_goodthomas_cost(
+    len: usize,
+    left_fft: &Arc<Recipe>,
+    right_fft: &Arc<Recipe>,
+    repeats: usize,
+) -> f32 {
+    let inners_cost = left_fft.cost(right_fft.len()) + right_fft.cost(left_fft.len());
+    let twiddle_cost = GOODTHOMAS_SLOPE * (len as f32).powf(GOODTHOMAS_EXP);
+    (inners_cost + twiddle_cost) * repeats as f32
+}
+
+const GOODTHOMASSMALL_SLOPE: f32 = 0.60873;
+const GOODTHOMASSMALL_EXP: f32 = 0.93289;
```

Comment:
Same here - these values are suggesting that GoodThomasSmall is strictly faster than GoodThomas, when that definitely isn't true.